src/mcts/node.h

```diff
@@ -382,8 +382,8 @@ class EdgeAndNode {
   }
 
   int GetVisitsToReachU(float target_score, float numerator, float default_q,
-                        bool logit_q) const {
-    const auto q = GetQ(default_q, logit_q);
+                        float drawscore, bool logit_q) const {
+    const auto q = GetQ(default_q, drawscore, logit_q);
```

Comment:
Uh, good find!.

..but that bug is bad. :-\
Actually, default arguments were only recently (2 years ago) allowed in the style guide, precisely because of this problem..

I suggest to remove the default value for `logit_q` and type that argument explicitly (as `/* logit_q= */ false`) at every place where it's called. There are many cases where default arguments improve readability, but in this case it does more harm than good.